.github/workflows/release_note.py

```diff
@@ -210,36 +212,95 @@ def generate_package_update_section(version):
                     output += _parse_match_section(s['matchBinaries'])
     return output
 
+def release_notes_changes_section():
+    return textwrap.dedent("""
+    ## Changes
+    The following packages have been upgraded, to address the mentioned CVEs:
+    **todo release facilitator: fill this in**
+    """)
 
-def create_github_release_notes(gardenlinux_version, commitish):
-    commitish_short=commitish[:8]
-    output = ""
-  
-    manifests = download_all_singles(gardenlinux_version, commitish_short)
-
-    output += generate_release_note_image_ids(manifests)
-
-    output += "\n"
+def release_notes_software_components_section(package_list):
+    output = "\n"
     output += "## Software Component Versions\n"
     output += "```"
     output += "\n"
+    packages_regex = re.compile(r'^linux-image-amd64$|^systemd$|^containerd$|^runc$|^curl$|^openssl$|^openssh-server$|^libc-bin$')
+    for entry in package_list.values():
+        if packages_regex.match(entry.deb_source):
+            output += f'{entry!r}\n'
+    output += "```"
+    output += "\n\n"
+    return output
+
+def release_notes_compare_package_versions_section(gardenlinux_version, package_list):
+    output = ""
+    version_components = gardenlinux_version.split('.')
+    # Assumes we always have version numbers like 1443.2
+    if (len(version_components) == 2):
+        try:
+            major = int(version_components[0])
+            patch = int(version_components[1])
+
+            if patch > 0:
+                previous_version = f"{major}.{patch - 1}"
+
+                output += f"## Changes in Package Versions Compared to {previous_version}\n"
+                output += "```diff\n"
+                output += subprocess.check_output(['/bin/bash','./hack/compare-apt-repo-versions.sh', previous_version, gardenlinux_version]).decode("utf-8")
```

Comment:
While I love that we have a diff right now, I think this output is also burdensome for the reader. They are difficult to comprehend, and you need to keep track if a change is just removing things or if it then adds the package again. Since we diff *every* package, we change, it's a lot of noise. Instead, we should have a more slim like:

Update:
- python 3.12.4 → 3.12.7. 
- linux 6.6.47 → 6.6.56

If someone want's to learn more about the product, we can link to the necessary package release, where all packages are listed. 

Not sure if this to much to be ask for?

Reply:
software developers are familiar with the diff format, and the purpose here is exactly to allow knowing all the changes in patch releases. purposefully, this diff is not done when we have a new major version because that would be huge.

this is now a discussion on the requirements but from my conversations with @pnpavlov and @gehoern I got that people actually want a full diff, so that's what is provided here.